influence_benchmark/RL/accelerate_slurm.yaml

```diff
@@ -1,3 +1,4 @@
+# Maybe put this in the normal config too?
```

Comment:
Is that easily possible with accelerate? I think I tried to do something like this before but accelerate got mad at me

Reply:
I think you can just pass everything as command line arguments as done in [this line](https://github.com/carolius/Influence-benchmark/blob/5df9ef5ae5e5da2db41fa7223e05c3153f3d7859/influence_benchmark/RL/base_iteration.py#L260)? Not sure why it would fail, as the self.accelerate_config_path ends up being formatted the same way as the line I sent when the command is sent to subprocess

Reply:
Well this just passes the config path in, not the config in?

Reply:
But yeah if it works then we should definitely do that